core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -85,14 +85,24 @@ object Automation {
             // NoFocus or Food/Growth Focus.
             if (surplusFood < 0)
                 yieldStats.food *= 8 // Starving, need Food, get to 0
-            else if (city.civ.getHappiness() > -1)
+            else if (city.civ.getHappiness() > 0)
                 yieldStats.food *= 2 //1.5f is preferred, but 2 provides more protection against badly configured personalities
-            else if (city.civ.getHappiness() < 0) {
+            else if (city.civ.getHappiness() < 9)
+                // Going below -10 happiness yields very severe penalties, let's try to avoid it
+                yieldStats.food *= 0
```

Comment:
Doesn't this cause Starvation?

Reply:
No, it is applied only when the no-starvation requirement on line 86-87 is met.